articles/quickstart/spa/_includes/_new_app_no_sample.html

```diff
@@ -1,5 +0,0 @@
-<% if (account.userName) { %>
```

Comment:
Why delete this file out of curiosity? It's possible that it's being used somewhere else.